contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,77 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+import logging
+from google import search
+
+def get_google_result(original_content):
+    search_keywords = original_content.split(' ', 1)[1]
+    if search_keywords == '--help':
+        help_message = '''
+                        To use this bot start message with @google followed
+                        by what you want to search for. If found, Zulip will
+                        return the first search result on google.
+                        '''
+        return help_message
+    else:
+        try:
+            urls = search(search_keywords, stop=20)
+        except:
+            logging.exception('no internet connection')
+            return "Error: search failed."
+
+        if not urls:
+            return "No URLs returned by google."
+
+        url = next(urls)
+
+        return 'Success: ' + str(url)
+
+class GoogleSearchHandler(object):
+    '''
+    This plugin allows users to enter a search
+    term in Zulip and get the top URL sent back
+    to the stream in which it was called.It looks
+    for messages starting with @google.
+    '''
+
+    def usage(self):
+        return '''
+            This plugin will allow users to search
+            for a given search term on Google from
+            Zulip. If mor help is needed with using
```

Comment:
`more` instead of `mor`, but maybe a rephrasing would be even better `use '@google --help' to get more information on the usage'.